docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -19,8 +19,8 @@ usernameless authentication for Teleport.
 - A web browser with WebAuthn support. To see if your browser supports
   WebAuthn, check the [WebAuthn
   Compatibility](https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/) page.
-- A signed and notarized version of `tsh` is required for Touch ID. This precludes
-  versions installed from Homebrew. [Download the macOS tsh installer](../../installation.mdx#macos).
+- A signed and notarized version of `tsh` is required for Touch ID. This means versions
+  installed from Homebrew will not work. [Download the macOS tsh installer](../../installation.mdx#macos).
```

Comment:
Do we want to mention self-compiled too?

Reply:
https://github.com/gravitational/teleport/pull/44943/commits/41cc1d739bd0702085e101290e94e903a49b982a